### ISSUE TITLE
Meditate sunrise/sunset

### ISSUE THREAD

**Issue by zedace** [rwtema/modjam2018#56]
Using Monk 1.4, I was in a 2x1 tunnel facing east with stone bricks all around, down level 12.  I achieved this advancement!

**Comment by darthvader45** [comment]
Yeah, might have to modify this one to have a minimum height of at least 62 and have no blocks above you.  Can't reasonably see the sunrise/sunset if underground or inside a house, right?
